src/styles/blocks/section-title.scss

```diff
@@ -0,0 +1,19 @@
+.section-title {
+  font-weight: 700;
+  font-size: 32px;
+  color: inherit;
+  letter-spacing: -1.5px;
```

Comment:
Consider using a variable for the letter-spacing to maintain consistency in design.